packages/framework/easy-coding-standard.yml

```diff
@@ -12,6 +12,7 @@ services:
                             - Shopsys\FrameworkBundle\Model
                             - Shopsys\FrameworkBundle\Component
                             - Shopsys\FrameworkBundle\Controller
+                            - Shopsys\FrameworkBundle\Form\Transformer
```

Comment:
According to the other changes do not you want to extend the analysis for `Shopsys\FrameworkBundle\Form\Admin\AdvancedSearch` too?

Reply:
good point, fixed